### PR TITLE
Fix safari table glitch

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Currently, there is a glitch in tables (tokens and neurons) in desktop Safari (see screenshots).

How to reproduce:
1. Open the tokens or neurons page.
2. Refresh the browser.

# Changes

- The issue is presumably connected to how Safari allocates extra spacing even when it might not be necessary. Replacing overflow-y: `auto` with `hidden` fixes it.

# Tests

- Manually with Safari, Chrome, Firefox.

# Screenshots

| Before | After |
|--------|--------|
| image | image | 
| image  | image |

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: frontend/src/lib/components/ui/ResponsiveTable.svelte [hunk 1]
```diff
@@ -185,7 +185,7 @@
 
     border-radius: var(--border-radius);
     // Otherwise the non-rounded corners of the header and last row would be visible.
-    overflow-y: auto;
+    overflow-y: hidden;
```
Comment (dskloetd) [inline frontend/src/lib/components/ui/ResponsiveTable.svelte:188]: Did you try if horizontal scrolling still works on the neurons table if the window is too small to fit the table?
Or do you have a demo I can try?

Comment (dskloetd) [reply frontend/src/lib/components/ui/ResponsiveTable.svelte:188]: I tried it. Seems to be fine.

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/components/ui/ResponsiveTable.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/ui/ResponsiveTable.svelte:188]: Thank you!

**Pull request merged**